src/boot/boot.c

```diff
@@ -1097,6 +1099,19 @@ static void config_defaults_load_from_file(Config *config, char *content) {
                         else
                                 log_error("Error parsing 'secure-boot-enroll-action' config option, ignoring: %s",
                                           value);
+                } else if (streq8(key, "secure-boot-enroll-timeout")) {
```

Comment:
in userspace we follow this rule to name config knobs so that they carry the unit as suffix, i.e. we usualy call timeout settings "FoobarSec" or so there, to make very clear that this is seconds, not something else. Now, loader.conf doesn't follow that rule, it has "timeout", which gives no such hint. But I'd prefer if for new knobs added we'd follow this rule too, i.e. name this "secure-boot-enroll-timeout-sec".

Reply:
renamed to `secure-boot-enroll-timeout-sec`, thanks!